include/blas_meta.h

```diff
@@ -190,6 +195,29 @@ struct is_sycl_scalar<float *> : std::false_type {};
 template <>
 struct is_sycl_scalar<double *> : std::false_type {};
 
+#ifdef BLAS_ENABLE_COMPLEX
+// SYCL Complex type alias
+template <typename T>
+using complex_sycl = typename cl::sycl::ext::oneapi::experimental::complex<T>;
+
+template <class type>
+struct is_complex_sycl
+    : std::integral_constant<bool,
+                             std::is_same_v<type, complex_sycl<double>> ||
+                                 std::is_same_v<type, complex_sycl<float>>> {};
+
+// STD Complex type alias
+template <typename T>
+using complex_std = typename std::complex<T>;
```

Comment:
It seems to me that redefine `std::complex` to `complex_std` doesn't have any particular utility. It is used only in `common_utils.hpp` and tests, but you can use  directly `std::complex`. Am I missing something? Do you have any particular reason in mind?

Reply:
Hmm yes indeed it looks like I forgot to reuse it in the tests (where i was planning to), and I'm reusing it heavily in benchmarks. Idk I was thinking about having it distinguishable from sycl::complex that's why. But yes I'll take note of it and discuss with the rest of the team if it's worth keeping!